src/EditorFeatures/Core/Shared/Tagging/EventSources/TaggerEventSources.ViewSpanChangedEventSource.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Editor.Tagging;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Tagging
+{
+    internal partial class TaggerEventSources
+    {
+        private class ViewSpanChangedEventSource : ITaggerEventSource
+        {
+            private readonly ForegroundThreadAffinitizedObject foregroundObject = new ForegroundThreadAffinitizedObject();
+
+            private readonly ITextView textView;
+            private readonly TaggerDelay textChangeDelay;
+            private readonly TaggerDelay scrollChangeDelay;
+
+            private Span? span;
+            private ITextSnapshot viewTextSnapshot;
+            private ITextSnapshot viewVisualSnapshot;
+
+            public event EventHandler<TaggerEventArgs> Changed;
+            public event EventHandler UIUpdatesPaused { add { } remove { } }
+            public event EventHandler UIUpdatesResumed { add { } remove { } }
```

Comment:
This is the reverse direction.  This is what the event source calls if it wants to pause UI updates.  The tagger attaches event handlers to this so it can be informed when the event source wants to pause the UI.

Because this event source never wants to pause the UI, it doesn't want these.  It just needs to be cause the ITaggerEventSource mandates it. (confusing i know)